CHANGES.md

```diff
@@ -3,6 +3,8 @@
 New Grammars:
 
 - added 3rd party Lang grammar to SUPPORTED_LANGUAGES [AdamRaichu][]
+- added 3rd party Phix grammar to SUPPORTED_LANGUAGES [PeteLomax][]
+- added 3rd party C3 grammar to SUPPORTED_LANGUAGES [aliaegik][]
```

Comment:
Is this unintended?

Reply:
No. Github Desktop insists I did that 13 days ago but it's far more likely something I tried 13 _months_ ago.